### PR TITLE
some requirments added

### PR DESCRIPTION
**Pull request by JDRanpariya**
it's cumbersome when applying runserver command and when it displays module not found

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ We are an open source, non-profit community, working to develop a democratic eng
...
 * **Citizen/Representative Engagement.** CiviWiki's core goal is to engage citizens and their representatives, with the goal of making government more accountable. CiviWiki will achieve this goal in two ways. First, CiviWiki will organize user's policy profile and compare it to every political candidate in the user's district. This quick, detailed, comparison will help users make informed votes, and we believe increased voter confidence will increase voter turnout. Second, CiviWiki will collect anonymized user data and forward district level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives' votes.
 
+##Requirements
+  #python modules
```
Comment (brylie) [inline README.md:16]: Use three hashmarks here (###) for proper nesting.

Comment (JDRanpariya) [reply README.md:16]: ok

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ We are an open source, non-profit community, working to develop a democratic eng
...
 * **Citizen/Representative Engagement.** CiviWiki's core goal is to engage citizens and their representatives, with the goal of making government more accountable. CiviWiki will achieve this goal in two ways. First, CiviWiki will organize user's policy profile and compare it to every political candidate in the user's district. This quick, detailed, comparison will help users make informed votes, and we believe increased voter confidence will increase voter turnout. Second, CiviWiki will collect anonymized user data and forward district level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives' votes.
 
-##Requirements
+###Requirements
```
Comment (brylie) [inline README.md:15]: Ah sorry. The headers should look like this:

```
## Requirements
### Python
```

Comment (JDRanpariya) [reply README.md:15]: Thank u

**Comment by brylie** [comment]
@AllContributors please add @JDRanpariya for documentation

**Pull request merged**